packages/testing-utils/types/index.d.ts

```diff
@@ -107,7 +107,8 @@ export interface SagaIteratorClone extends SagaIterator {
 export function createMockTask(): MockTask
 
 export interface MockTask extends Task {
-  setRunning(running: boolean): void
+  setRunning(): void
   setResult(result: any): void
   setError(error: any): void
+  setAborted(): void
```

Comment:
On the second thought - `setAborted` is somewhat synonymous to `setError(err?: any)`. Thoughts?

Reply:
Agreed, I don't think setAborted is needed. The equivalent to the saga behavior API is generator.throw, so having just setError makes sense to me.